docs/api-reference.md

```diff
@@ -235,8 +225,12 @@ Example:
 ### ConnectionNormalizer and ConnectionMatcher JavaScript files
 The JavaScript files for connection normalizer and connection matcher are deprecated as of Tableau 2020.3. In Tableau 2021.2 support was removed and an error will occur when loading the connector. The element `<script file="fileName.js"/>`, which was added inside the `<connection-matcher>` and `<connection-normalizer>` element, and the `<connection-matcher>` element itself, are the deprecated APIs. The `<connection-normalizer>` element is still supported in the connectionResolver.tdr file as shown in the connection-normalizer section above.
 
-### SetImpersonateAttributes connection helper
+### `SetImpersonateAttributes` connection helper
 This connection helper is deprecated as of Tableau 2020.1, since we always set impersonate attributes for all connectors. Trying to use this in a JavaScript component will throw an error when attempting to connect.
 
-### <setImpersonateAttributes/> XML tag
-This xml tag is deprecated as of Tableau 2020.1, though it has not yet been removed from the XSD. Since we always set this property starting with 2020.1, this tag is redundant.
+### `<setImpersonateAttributes/>` XML tag
+`<setImpersonateAttributes/>` xml tag is deprecated as of Tableau 2020.1, though it has not yet been removed from the XSD. Since we always set this property starting with 2020.1, this tag is redundant.
+
+### Logging
+`logging.Log()` JavaScript function is deprecated.  Any call to this function will log message "Connector SDK logging.log() function is deprecated." in the log file.
```

Comment:
Can you add which versions this method is deprecated in? Since we backported this to earlier versions.